tests/test_galaxycluster.py

```diff
@@ -1,12 +1,12 @@
 """
 Tests for datatype and galaxycluster
 """
+import numpy as np
```

Comment:
New numpy import, but not needed anywhere in `test_galaxycluster.py` Please remove.

Reply:
done